CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.2.1] - 2022-03-16
+
+### Added
+- The SPIRE Agent `fetch jwt` CLI command now supports JSON output (#2650)
+
+### Changed
+- OIDC Discovery Provider now includes the `alg` parameter in JWKs to increase compatibility  (#2771)
+- SPIRE Server logs additional authorization information with RPC requests (#2776)
+- Small documentation improvements (#2746, #2792)
+
+### Fixed
+- SPIRE Server now properly rotates signing keys when prepared or activated keys are lost from the database (#2770)
+- The AWS IID node attestor now works with instance profiles which have paths (#2825)
+- Fixed a crash in SPIRE Agent caused by a race on the agent cache (#2699)
+- SPIRE Server waits for plugin requests to finish during graceful shutdown (#2722)
```

Comment:
I would probably move this to the _Changed_ category since this is more a change in the behavior than a fix to a specific issue?
The description may also provide the idea that we are already doing a graceful shutdown, but that's the change that is being introduced.

Maybe something like this works better?
```suggestion
- SPIRE Server now gracefully stops plugin servers, allowing outstanding RPCs a chance to complete (#2722)
```